buildbot/make_status_file_table.py

```
@@ -74,7 +74,7 @@ def path_to_dict(path):
 
         flag = "//%Impl status : "
 
-        for l in f.readlines():
+        for l in f:
```

Ruff (0.16.0)

[error] Ambiguous variable name: `l`

(E741)